src/derivatives/imaging.md

```diff
@@ -406,8 +410,9 @@ Template:
 ```Text
 <pipeline_name>/
     sub-<label>/
-        anat/
-            <source_entities>[_hemi-{L|R}][_space-<space>][_atlas-<label>][_res-<label>][_den-<label>]_dseg.{label.gii|dlabel.nii}
+        [ses-<label>/]
+            anat/
+                <source_entities>[_hemi-{L|R}][_space-<space>][_atlas-<label>][_res-<label>][_den-<label>][_desc-<label>]_dseg.{label.gii|dlabel.nii}
```

Comment:
I don't think `res` is applicable to gifti files, though it _is_ applicable to cifti files.

Reply:
```suggestion
                <source_entities>[_hemi-{L|R}][_space-<space>][_atlas-<label>][_den-<label>][_desc-<label>]_dseg.label.gii
                <source_entities>[_hemi-{L|R}][_space-<space>][_atlas-<label>][_res-<label>][_den-<label>][_desc-<label>]_dseg.dlabel.nii
```

Reply:
Do you have any thoughts on how to represent in the template the fact that accompanying `.json` and `.tsv` files are allowed? Three options that spring to mind:
1. Include these file extension options on _both_ lines, that is:

    ```
    ..._dseg.{label.gii|json|tsv}
    ..._dseg.{dlabel.nii|json|tsv}
    ```

    I think this is a bit clearer, but technically redundant.
2. Include the file extension options only on the more permissive line, that is:

    ```
    ..._dseg.{label.gii}
    ..._dseg.{dlabel.nii|json|tsv}
    ```

    Maybe this would be confusing.
3. Have a separate line for each of `label.gii`, `dlabel.nii`, `json`, `tsv`, I guess forbidding `res` only for `dlabel.nii`?